daq/runner.py

```diff
@@ -622,6 +627,10 @@ def _target_set_trigger(self, device, remote_trigger=False):
             LOGGER.debug('Target device %s already queued', device)
             return
 
+        if device.mac in self._target_set_block:
```

Comment:
I think something like target_set_triggered will be better than target_set_block(it raises a new concept of device blocking which confuses me at first)

Reply:
The intent was that "triggered" means that it's been triggered for a test run, while "blocked" is to mean that it's blocked from future test runs. In single-shot mode, triggering causes it to be blocked, but there's cases where triggered != blocked, and also where we might want to pre-block devices so they are never triggered.  So yes, it was intentionally a new concept... not sure how to make it less confusing.

Reply:
What's the case where blocked != triggered under single shot? The variable can be scoped to be single_shot_target_set_triggered. There isn't a way to pre-block devices either so I don't think we need to introduce this "blocked" concept at the moment.

Reply:
THe concept there either way ("this is a set of devices that should not be tested") -- it's just a matter of what we call it.  I don't want to specifically scope it to "single shot" since that's mixing the use (single shot) with the behavior (prevent device). Would block_devices or prevent_devices be better?  What would you call something that was just meant to be a "set of devices that shouldn't be tested"?